### PR TITLE
Update import statements

### PR DESCRIPTION
**Pull request by jansunil**
Update import statements in python scripts for AD4170 and AD7134

### CODE REVIEW

File: projects/ad7134_iio/tests/func/test_ad7134_example.py [hunk 1]
```diff
@@ -5,8 +5,7 @@
 import csv
 import os
 from numpy import ndarray
-#import adi.ad7134      # TODO - Enable once ad7134 iio linux drivers are publicly released
-from ad7134 import *    # TODO - Delete later
+from adi.ad7134 import *    # TODO - Delete later
```
Comment (SaikiranGudla) [inline projects/ad7134_iio/tests/func/test_ad7134_example.py:8]: Is this TODO required ?

Comment (jansunil) [reply projects/ad7134_iio/tests/func/test_ad7134_example.py]: Not needed, removed this

**Comment by jansunil** [comment]
@ribdp @SaikiranGudla - Are there any more review comments on this PR?

**Comment by jansunil** [comment]
@SaikiranGudla @ribdp - Does this PR look good to go?

**Review by ribdp — APPROVED** [review]

**Review by SaikiranGudla — APPROVED** [review]

**Pull request merged**